packages/peregrine/lib/talons/Region/useRegion.js

```diff
@@ -22,7 +22,7 @@ export const useRegion = props => {
         countryCodeField = 'country',
         fieldInput = 'region',
         fieldSelect = 'region',
-        optionValueKey = 'code',
+        optionValueKey = 'id',
```

Comment:
With the usage of aliases above, I assumed we were aiming for a backwards compatible fix, but I believe this would be breaking. I would change back and just make sure our component usage passes `id` here. When we get to a breaking change release line we can likely remove this logic entirely.